packages/app-mobile/utils/hooks/useKeyboardState.ts

```
@@ -44,4 +44,4 @@ const useKeyboardState = () => {
 	}, [keyboardVisible, hasSoftwareKeyboard, isFloatingKeyboard, keyboardHeight]);
 };
 
-export default useKeyboardState;
+export default useKeyboardState;
```

ESLint

[error] Newline required at end of file but not found.

(eol-last)